modules/configure-google-cloud-scc.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-google-cloud-scc.adoc
+:_module-type: PROCEDURE
+[id="configure-google-cloud-scc_{context}"]
+= Configuring Google Cloud SCC
+
+Start by adding {product-title} as a trusted Cloud SCC source.
+
+.Procedure
+. Follow the link:https://cloud.google.com/security-command-center/docs/how-to-security-sources[Adding vulnerability and threat sources to Cloud Security Command Center] guide and add {product-title} as a trusted Cloud SCC source.
+Note down the *Source ID* that Google Cloud creates for your {product-title} integration.
```

Comment:
I don't think "Note down" is the right phrase. I'd say just "Note" or "Make a note of"